wordpress-mobile/WordPress-iOS
# [Notifications P1] Replace Notifications Cell Content

Fixes #22465 | #22472

## Description
Utilises the newly added `NotificationsTableViewCellContent` to replace the old cells used in NotificationsViewController.
Also contains improvements to `AvatarsView` like placeholder image and parsing multiple image urls from the `Notification` response.

## Testing Steps
1. Install & login to Jetpack App
2. Navigate to `Notifications` tab.
3. ✅ Verify that the UI matches the designs.
4. ✅ Verify, cell selection does not highlight the cell.
5. ✅ Verify various avatar styles: single, double, triple.
6. ✅ Retry steps 3, 4 and 5 after toggling the theme color.

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:


Testing checklist:
